src/caretogether-pwa/src/Components/Volunteers/VolunteerApproval.tsx

```diff
@@ -207,8 +207,8 @@ function VolunteerApproval(props: { onOpen: () => void }) {
   // Filter volunteer families by name and by applicable roles.
   const filteredVolunteerFamilies = volunteerFamilies.filter(family => (
       filterText.length === 0 ||
-      family.family?.adults?.some(adult => simplify(`${adult.item1?.firstName} ${adult.item1?.lastName}`).includes(filterText)) ||
-      family.family?.children?.some(child => simplify(`${child?.firstName} ${child?.lastName}`).includes(filterText))) && (
+      family.family?.adults?.some(adult => simplify(`${adult.item1?.firstName} ${adult.item1?.lastName}`).includes(filterText.toLowerCase())) ||
```

Comment:
This fix is correct. Can you please also add the fix to the `VolunteerProgress` screen, which the bug also applies to?

Reply:
Yep, I actually added them in PR #452 .
Sorry for making this PR spam.
Actually, I am new to open-source so was a bit confused.